examples/real-world/containers/RepoPage.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { loadRepo, loadStargazers } from '../actions';
+import Repo from '../components/Repo';
+import User from '../components/User';
+import List from '../components/List';
+
+function loadData(props) {
+  const { fullName } = props;
+  props.loadRepo(fullName, ['description']);
+  props.loadStargazers(fullName);
+}
+
+class RepoPage extends Component {
+  constructor(props) {
+    super(props);
+    this.renderUser = this.renderUser.bind(this);
+    this.handleLoadMoreClick = this.handleLoadMoreClick.bind(this);
+  }
+
+  componentWillMount() {
+    loadData(this.props);
```

Comment:
Thanks @gaearon! I actually tried to follow FSA (using `meta`); once the ["sequence" stuff](https://github.com/acdlite/flux-standard-action/issues/7) gets figured out maybe I can change the API to support that.

If you get a chance, I'd love your opinion on [a better way to "cancel" entire Action Creator sequences](https://github.com/idolize/redux-requests/issues/1) via middleware (or something else)